src/plugins/data/common/search/search_source/search_source.test.ts

```diff
@@ -824,7 +825,7 @@ describe('SearchSource', () => {
     test('should serialize filters', () => {
       const filter = [
         {
-          query: 'query',
+          query: { q: 'query' },
```

Comment:
Query used to be any, and now it's an object.
Only a few tests used this as a string, and as defining `query` to be `string | Record` made casting more complex everywhere, I decided to leave it as an object only for now.